npm/ng-packs/angular.json

```
@@ -366,6 +366,29 @@
         }
       }
     },
+    "schematics": {
+      "projectType": "library",
+      "root": "packages/schematics",
+      "sourceRoot": "packages/schematics/src",
+      "prefix": "abp",
+      "architect": {
+        "test": {
```

```suggestion
      "architect": {
        "build": {
          // do not remove the build and options properties, the symlink-manager process breaks without these
          "options": {
            "tsConfig": "packages/commercial-ui/tsconfig.lib.json",
            "project": "packages/commercial-ui/ng-package.json"
          }
        },
        "test": {
```

If the `build` property with the `options` property does not exist, the symlink-manager breaks